### ISSUE TITLE
S16 - More models support + Local LLMs

### ISSUE THREAD

**Issue by RamiAwar** [RamiAwar/dataline#267]
### Privileged issue

- [X] I'm @ramiawar or he asked me directly to create an issue here.

**Comment by RamiAwar** [comment]
Update: Let's do this later maybe...

<img width="625" alt="image" src="https://github.com/user-attachments/assets/b05a23ba-9ff1-4520-949a-554a40807eef">

**Comment by symbarh** [comment]
Is there any progress on this?

**Comment by RamiAwar** [comment]
Nope, project is paused for now @symbarh . I'm willing to review PRs but don't have time to work on it currently.

**Comment by NPCRUS** [comment]
maybe at least opnapi api compatible backends, like deepseek, with the ability to set model name?

**Comment by RamiAwar** [comment]
That's already possible @NPCRUS
